doc/wiki/How_To.md

```diff
@@ -0,0 +1,111 @@
+# How to?
+## This document should help you as a user to understand all functionalities of the application.
+
+##Introduction
+After logging in you will be navigated to the dashboard screen.
+Here you see the navigation at the left border and some fancy dashboards in the content area.
+They give you some information about children and can be defined by you, but later more about that (Verlinkung section Dashboard).
```

Comment:
`(Verlinkung section Dashboard)`

Reply:
There is no dashboard section right now. Should create a new issue for that.